workshops/workshop_games_rental_dev_team_en.md

```diff
@@ -0,0 +1,21 @@
+# Event Storming Workshop - Games Rental
+
+## Development Team
+
+You work in a software house, that has been asked to create the IT system for games rental company. Project is pretty bit and it requires involvement of a few people in the development team. We received a great information, that there is great, highly involved domain expert on customer’s side. He know the business for the game rentals very well. He is available for us for the event storming session.
+
+Sales department provided us some general information about the scope, that needs to be implemented. However, the deadline is very close so we need to start coding it as soon as possible.
+
+Cooperate with your team. Ask the domain expert about all elements in the process and all clarify all the considerations. Treat him like a partner, a part of your team, that needs to create the system. Do not hate him and do not act in negative manner. Tell him when he provides valuable insights, so he can feel more comfortable and valuable in the session. This will help in getting more information from him.
```

Comment:
```suggestion
Cooperate with your team. Ask the domain expert about all elements in the process and clarify all the considerations. Treat him like a partner, a part of your team, that needs to create the system. Do not hate him and do not act in negative manner. Tell him when he provides valuable insights, so he can feel more comfortable and valuable in the session. This will help in getting more information from him.
```